tests/components/flipr/test_sensor.py

```diff
@@ -84,3 +86,31 @@ async def test_sensors(hass: HomeAssistant) -> None:
     assert state.attributes.get(ATTR_ICON) == "mdi:pool"
     assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == "mV"
     assert state.state == "0.23654886"
+
+
+async def test_error_flipr_api_sensors(hass: HomeAssistant) -> None:
+    """Test the Flipr sensors error."""
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        unique_id="test_entry_unique_id",
+        data={
+            CONF_EMAIL: "toto@toto.com",
+            CONF_PASSWORD: "myPassword",
+            CONF_FLIPR_ID: "myfliprid",
+        },
+    )
+
+    entry.add_to_hass(hass)
+
+    registry = await hass.helpers.entity_registry.async_get_registry()
```

Comment:
Please import the entity_registry module as normal at the top of the module and then get the registry instance with `async_get`. Accessing `hass.helpers` is legacy.

Reply:
Done via PR https://github.com/home-assistant/core/pull/67477